src/SmartComponents/CompletedTaskDetails/CompletedTaskDetails.js

```diff
@@ -291,22 +289,24 @@ const CompletedTaskDetails = () => {
                   ouiaId={`${completedTaskDetails.id}-completed-jobs-table`}
                   columns={isConversionTask() ? conversionColumns : columns}
                   items={completedTaskJobs}
-                  filters={buildFilterConfig()}
-                  options={{
-                    ...TASKS_TABLE_DEFAULTS,
-                    exportable: {
-                      ...TASKS_TABLE_DEFAULTS.exportable,
-                      columns: isConversionTask()
-                        ? conversionColumns
-                        : exportableColumns,
-                    },
-                    detailsComponent: completedTaskJobs.some((job) =>
-                      hasDetails(job)
-                    )
-                      ? JobResultsRow
-                      : undefined,
-                    actionResolver,
-                  }}
+                  filters={!tableLoading && buildFilterConfig()}
```

Comment:
Instead of hiding the filter, should we try to fix the root cause? This [line](https://github.com/RedHatInsights/tasks-frontend/blob/0d0c6740077b7af0a3d406f1e0c3a5e5df11ddbb/src/SmartComponents/CompletedTaskDetails/Filters.js#L10) is making the trouble, proper handling of empty job should do the trick.

Reply:
Done ... again :)
![Screenshot from 2024-04-03 20-52-34](https://github.com/RedHatInsights/tasks-frontend/assets/4008744/9484aa04-2802-4977-b2e3-6cb840d493ac)

Reply:
Although hiding the filter works too, because there aren't any tasks/jobs to filter at that stage anyway.